package.json

```diff
@@ -102,8 +102,8 @@
     "storybook-addon-jsx": "^7.1.0"
   },
   "dependencies": {
-    "futil-js": "^1.58.0",
-    "lodash": "^4.17.4",
+    "futil-js": "^1.61.0",
```

Comment:
necessary for `mergeOverAll` (added in futil@1.60.0)

Reply:
we should actually swap that out for the shinier `futil` instead of `futil-js`